routes/donations.js

```diff
@@ -119,6 +125,9 @@ router.post('/', async (req, res) => {
       )
       RETURNING *;`,
       {
+        id,
+        routeId,
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'routeId' is not defined.